apps/jurisdiction/migrations/0001_initial.py

```diff
@@ -54,6 +54,6 @@ class Migration(migrations.Migration):
         migrations.AddField(
             model_name='jurisdiction',
             name='state',
-            field=models.ForeignKey(to='jurisdiction.State'),
+            field=models.ForeignKey(to='jurisdiction.State', on_delete=models.CASCADE),
```

Comment:
Can you tell me a little bi about why use `CASCADE` here. Makes me a little nervous.

Reply:
Required argument since Django 2

Reply:
Gotcha!

Note to my future self here: https://code.djangoproject.com/ticket/28677